CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added runtime option to auto spawn the ASpatialMetricsDisplay for remote debugging of server metrics - bEnableMetricsDisplay.
 - Added runtime option to batch spatial position updates to runtime.
 - Added runtime option to pack unreliable RPCs from the same frame in a single component update to save bandwidth - bPackUnreliableRPCs.
+- Added `USpatialstatics` helper function blueprint library.
```

Comment:
I think a more helpful change log is describing what kinds of functions you've added

Reply:
Also this isn't a great name. I kept reading "statistics". "Statics" doesn't really tell you anything about what it is.

Reply:
Intention was to mirror the engine concept of `GameplayStatics` http://api.unrealengine.com/INT/API/Runtime/Engine/Kismet/UGameplayStatics/

Will expand on CHANGELOG yea

Reply:
```suggestion
- Added helper methods for Offloading to new `USpatialStatics` blueprint function library.
```

Reply:
Changed this myself at same time @mattyoung-improbable, can you TAL? 

@samiwh are you okay with this being a parallel of gameplay statics or still think it's confusing?

Reply:
If users are used to that naming and that they expect it then sure. But if we don't think that then we shouldn't use a bad name because someone else used a bad name.